boefjes/boefjes/plugins/kat_rdns/boefje.json

```diff
@@ -0,0 +1,15 @@
+{
+    "id": "rdns",
+    "name": "Reverse DNS",
+    "description": "Resolve IP-addreses to a hostname",
+    "consumes": [
+        "IPAddressV4",
+        "IPAddressV6",
+        "IPV4NetBlock",
+        "IPV6NetBlock"
```

Comment:
This shouldn't include IPV6NetBlock because resolving all hosts in an IPv6 netblock isn't possible.

Reply:
I removed it from boefje.json as suggested.

Reply:
I'd like to ervert this change, ipv6 ranges should be included, but similar to ipv4 ranges, they should only return the rdns directly related to the range itself, not all the included ip's